package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -314,13 +494,42 @@ class WaterBridgeAnalysis(HydrogenBondAnalysis):
 
     .. versionadded:: 0.17.0
     """
+    # use tuple(set()) here so that one can just copy&paste names from the
+    # table; set() takes care for removing duplicates. At the end the
+    # DEFAULT_DONORS and DEFAULT_ACCEPTORS should simply be tuples.
+
+    #: default heavy atom names whose hydrogens are treated as *donors*
+    #: (see :ref:`Default atom names for hydrogen bonding analysis`);
+    #: use the keyword `donors` to add a list of additional donor names.
+    DEFAULT_DONORS = {
+        'CHARMM27': tuple(set([
+            'N', 'OH2', 'OW', 'NE', 'NH1', 'NH2', 'ND2', 'SG', 'NE2', 'ND1', 'NZ', 'OG', 'OG1', 'NE1', 'OH'])),
+        'GLYCAM06': tuple(set(['N', 'NT', 'N3', 'OH', 'OW'])),
+        'other': tuple(set([]))}
+
+    #: default atom names that are treated as hydrogen *acceptors*
+    #: (see :ref:`Default atom names for hydrogen bonding analysis`);
+    #: use the keyword `acceptors` to add a list of additional acceptor names.
+    DEFAULT_ACCEPTORS = {
+        'CHARMM27': tuple(set([
+            'O', 'OC1', 'OC2', 'OH2', 'OW', 'OD1', 'OD2', 'SG', 'OE1', 'OE1', 'OE2', 'ND1', 'NE2', 'SD', 'OG', 'OG1', 'OH'])),
+        'GLYCAM06': tuple(set(['N', 'NT', 'O', 'O2', 'OH', 'OS', 'OW', 'OY', 'SM'])),
+        'other': tuple(set([]))}
+
+    #: A :class:`collections.defaultdict` of covalent radii of common donors
+    #: (used in :meth`_get_bonded_hydrogens_list` to check if a hydrogen is
+    #: sufficiently close to its donor heavy atom). Values are stored for
+    #: N, O, P, and S. Any other heavy atoms are assumed to have hydrogens
+    #: covalently bound at a maximum distance of 1.5 Å.
+    r_cov = defaultdict(lambda: 1.5,  # default value
```

Comment:
Should also be shared between here and HBA

Reply:
Old HBA removed.